kuma/static/js/experiment-static-examples-on-top.js

```diff
@@ -0,0 +1,14 @@
+(function() {
```

Comment:
Yep, those could both use it. (Told you we miss it all the time. Sigh.)

Should `experiment-wiki-content.js` still live on? It was initially a created as a stub for the experiment you're starting in this PR. It's not running or anything, so maybe it can be dropped?